meta/labs_program/interview.md

```diff
@@ -0,0 +1,16 @@
+Explain what labs program is.
+
+1. Tell me about your extracurriculars. What does your day look like after school ends? Can you rank them in order of priority?
+2. So you have a bunch of shit going on. Why do you care about this? 
+3. Imagine you are someone walking into your club for the first time. What's that experience like? How does someone find out about your club? What are the biggest problems you foresee running into?
+4. If your club goes really well, what will that look like?
+5. Why do you want to be part of the labs program? Can you spend 5 hours outside of your club a week?
+
+Club fair?
+First club meeting?
```

Comment:
Quick note: Markdown will put these two lines on the same line when rendered because it thinks they're part of the same paragraph. Here's what it'll look like:

![tmp](https://cloud.githubusercontent.com/assets/992248/9365441/70162052-4666-11e5-9712-0ec6f1564b6e.png)

The way to fix this is to add two spaces to the end of line 9 to indicate you want the lines to be separate (I know, I know). Example:

```
Club fair?<SPC><SPC>
First club meeting?
```